### DISCUSSION TITLE
MenuDesigner Pro

### DISCUSSION THREAD

**Discussion by lscheffler** [VFPX/GoFish#141] [Ideas]
Using MenuDesigner Pro to open mnx on the item of the hit. This is just a stub.

Anyway, do you like the idea?

**Comment by DougHennig** [comment]
It should be an option the user can turn on if MenuDesigner Pro is installed.

**Reply by lscheffler** [reply]
Something the like
- one need to figure out if installed
- one can check if MenuDesigner Pro is toggled on for project manager via the code from Ricks tool
- and now one can preset the option

**Comment by lscheffler** [comment]
@DougHennig Is there an URL for MenuDesigner Pro. I miss it in the session papers.

**Comment by DougHennig** [comment]
It hasn't been added to VFPX yet. Soon I think.

**Reply by lscheffler** [reply]
Oh ok. We heared Rick about it, I guess. ;)
